e2etest/config.go

```diff
@@ -0,0 +1,73 @@
+// Copyright © Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package e2etest
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/JeffreyRichter/enum/enum"
+)
+
+// clearly define all the inputs to the end-to-end tests
+// it's ok to panic if the inputs are absolutely required
+// the general guidance is to take in as few parameters as possible
+type GlobalInputManager struct{}
+
+func (GlobalInputManager) GetAccountAndKey(accountType AccountType) (string, string) {
+	var name, key string
+
+	switch accountType {
+	case EAccountType.Standard():
+		name = os.Getenv("AZCOPY_E2E_ACCOUNT_NAME")
+		key = os.Getenv("AZCOPY_E2E_ACCOUNT_KEY")
+	default:
+		panic("Only the standard account type is supported for the moment.")
```

Comment:
Would we end up with 3 key pairs in the end: standard, another pair for a HNS-enabled account, and S3?